### ISSUE TITLE
memcached client does not allow arguments to stats

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3661]
|<img alt="dlsspy's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| dlsspy reported|
|-|-|
|Trac ID|trac#3661|
|Type|enhancement|
|Created|2009-02-18 19:44:57Z|
|Branch|https://github.com/twisted/twisted/tree/memcache-stats-arg-3661|

The stats command takes arguments to get to various sets of stats, but the interface does not support this.

Attachments:

* [memcached-stats-fix.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/716/716c0e350e798d7347817743da36482ff61c84ad/8acea13c59df22e773f72d422b9c64de923611cd.diff) (1544 bytes) - added by dlsspy on 2009-02-18 19:45:27Z - Allow arguments to stats.

<details><summary>Searchable metadata</summary>

```
trac-id__3661 3661
type__enhancement enhancement
reporter__dlsspy dlsspy
priority__normal normal
milestone__ 
branch__branches_memcache_stats_arg_3661 branches/memcache-stats-arg-3661
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__memcached memcached
time__1234986297000000 1234986297000000
changetime__1236123006000000 1236123006000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [26332]) Branching to 'memcache-stats-arg-3661'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Mostly looks good.  I'm going to fix a few coding standard issues and other minor things.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @exarkun|
|-|-|

This protocol is lame... please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [26335]) Merge memcache-stats-arg-3661

Author: dlsspy, exarkun
Reviewer: therve
Fixes: [#3661](https://github.com/twisted/twisted/issues/3661)

Add support for the optional `args` parameter to the MemCache client `stats`.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
